k-distribution/src/main/scripts/lib/pyk/pyk/kast.py

```diff
@@ -619,8 +613,8 @@ def __init__(self, new_sort: KSort, old_sort: KSort, att=EMPTY_ATT):
     def from_dict(cls: Type['KSortSynonym'], d: Dict[str, Any]) -> 'KSortSynonym':
         cls._check_node(d)
         return KSortSynonym(
-            new_sort=KSort.from_dict(d['new_sort']),
-            old_sort=KSort.from_dict(d['old_sort']),
+            new_sort=KSort.from_dict(d['newSort']),
+            old_sort=KSort.from_dict(d['oldSort']),
```

Comment:
I want to see a test for this one. Encoding/decoding should eventually be thoroughly tested.